en/config_mc/filter_tuning.md

```diff
@@ -0,0 +1,98 @@
+# MC Filter Tuning & Control Latency
+
+Filters can be used to trade off [control latency](#control-latency), which affects flight performance, and noise filtering, which impacts both flight performance and motor health.
+
+This topic provides an overview of control latency and PX4 filter tuning.
+
+:::note
+Before filter tuning you should do a first pass at [Basic MC PID tuning](../config_mc/pid_tuning_guide_multicopter_basic.md).
+The vehicle needs to be undertuned (the **P** and **D** gains should be set too low), such that there are no oscillations from the controller that could be interpreted as noise (the default gains might be good enough).
+:::
+
+## Control Latency
+
+The *control latency* is the delay from a physical disturbance of the vehicle until the motors react to the change.
+
+:::tip
+Lowering latency allows you to increase the rate **P** gains, which results in better flight performance.
+Even one millisecond difference in the latency can have a significant impact.
+:::
+
+The following factors affect control latency:
+- A soft airframe or soft vibration mounting increases latency (they act as a filter).
+- Low-pass filters in software and on the sensor chip trade off increased latency for improved noise filtering.
+- PX4 software internals: the sensor signals need to be read in the driver and then pass through the controller to the output driver.
```

Comment:
Here we can also mention the update rate, configured with `IMU_GYRO_RATEMAX`

Reply:
@bkueng So does a higher rate reduce latency? Any reason not to use the highest rate available? / set to 0 and use the sensor default rate?

Reply:
Yes latency. It is really CPU intense, you quickly hit 100%. 2khz is pretty much the limit, on h7 possibly 4khz.

Reply:
The danger is if you set the rate too high it will starve things like mavlink and you won't be able to communicate with the board. This is why the param metadata max is 2 kHz (but you can override it).

As a precaution you could mention something simple like 4 kHz+ only recommended for stm32h7 or newer.

Reply:
Thanks. I've added this note.